app/views/refinery/_footer.html.erb

```diff
@@ -20,10 +20,10 @@
     </div>
   </div>
   <div class="footer-bottom">
-    <div class="footer-mapc-brand">
+    <%= link_to 'https://www.mapc.org/', class: 'footer-mapc-brand' do %>
```

Comment:
I know that this is at least a faux-pas in other stacks: don't use link helpers when linking externally. 

Is this a thing in Rails, too?

Reply:
Folks in Rails land seem to lack feelings about this. Both are familiar enough that there isn't any advantage to going one way or the other.